src/main/resources/db/migration/V005__Security.sql

```diff
@@ -0,0 +1,12 @@
+alter table migrations.user add column password text NOT NULL;      -- Table must be truncated before deployment
```

Comment:
I think I will modify this table to have a PK(email) instead, this should gurantee that no duplicate emails are used for registry.
At the same time, the current work I have for the auth controller does check the database for an email match.

Thoughts?

Reply:
i say use emails